conans/client/manager.py

```diff
@@ -449,7 +450,7 @@ def local_package(self, package_folder, conanfile_path, build_folder, source_fol
         output = ScopedOutput("PROJECT", self._user_io.out)
         conanfile = self._load_consumer_conanfile(conanfile_path, install_folder, output,
                                                   deps_info_required=True)
-        packager.create_package(conanfile, source_folder, build_folder, package_folder,
+        packager.create_package(conanfile, None, source_folder, build_folder, package_folder,
```

Comment:
Here, instead of None, provide the "id" of the package, if local then ``os.path.basename(package_folder)``